test/test_helper.rb

```diff
@@ -1,6 +1,7 @@
 #$LOAD_PATH.unshift File.expand_path("../lib", __dir__)
 ENV["RAILS_ENV"] = "test"
 
+require "active_record"
```

Comment:
If we move `require "active_snapshot/config"` outside of the `on_load` then can we avoid this extra require?